trio/_signals.py

```diff
@@ -139,14 +139,14 @@ def open_signal_receiver(*signals):
 
     Example:
```

Comment:
A few English nits:
- Comma after "program"
- Capitalize "Windows"
- This is really about _receiving_ signals, not sending them. I would say "can be handled" or "can be received."
- Very minor nit: Maybe better to avoid the passive voice here. Trio docs seem to aim for "friendly second-person style."

I've left concrete suggestions inline